### DISCUSSION TITLE
[Feature] Clicking the Back button should restore the previous scroll position

### DISCUSSION THREAD

**Discussion by Germs2004** [immich-app/immich#12884] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

This is for the web version of Immich:

In the main "Photos" view, this already works.  If I scroll down halfway in the list, click a photo, then click the provided back arrow button, Immich puts me back halfway down the list again, so I don't lose my position.  (Although, it technically does scroll me slightly so the photo I came from is scrolled to the top of my view rather than exactly where it was scrolled previously).

I request that feature also be made to work in the "People" view.  Currently, if I scroll halfway down and click a person to view their album, then click the back arrow button, it takes me back to the top of the People view again, so I have to scroll back down each time to find where I was.

Also, a minor consistency issue: the tooltip of the back arrow in the Photos view says "Go back" but the tooltip in the People view says "Close".

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by Acidiphilium** [comment]
Same thing happens when you're viewing a picture on an album then go back, you're again at the top of the that album. Would be amazing to go back with same exact scrolling position.

**Comment by bo0tzz** [comment]
#3114
